docs/installation-aks.md

```diff
@@ -0,0 +1,407 @@
+```
+
+This will create three deployments: etcd-operator, etcd-backup-operator, and restore-operator. This will also create three new Custom Resource Definitions: `etcdclusters.etcd.database.coreos.com`, `etcdbackups.etcd.database.coreos.com` and `etcdrestores.etcd.database.coreos.com`. These can be used to create an etcd cluster, along with backup and restore operations. 
+
+#### Create etcd Cluster
+
+Once etcd Operator has been installed, you can create a cluster. For production scenarios, we recommend a three node cluster. For convenience purposes, we have provided a Helm chart that will create a cluster and configure a Kubernetes CronJob to enable automatic backups of the cluster.
+
+**Bash**
+```console
+helm repo add azure https://kubernetescharts.blob.core.windows.net/azure
+helm install azure/svc-cat-etcd --set azure.storage.account=$AZURE_STORAGE_ACCOUNT --set azure.storage.key=$AZURE_STORAGE_KEY
+```
+
+**PowerShell**
+```console
+helm repo add azure https://kubernetescharts.blob.core.windows.net/azure
+helm install azure/svc-cat-etcd --set azure.storage.account=$env:AZURE_STORAGE_ACCOUNT --set azure.storage.key=$env:AZURE_STORAGE_KEY
+```
+
+Once completed, you should see several etcd pods running:
+
+```console
+$ kubectl get pods
+
+NAME                                                              READY     STATUS    RESTARTS   AGE
+etcd-operator-etcd-operator-etcd-backup-operator-6b697d96c95fgv   1/1       Running   0          11m
+etcd-operator-etcd-operator-etcd-operator-676764c476-n4ftv        1/1       Running   0          11m
+etcd-operator-etcd-operator-etcd-restore-operator-7c8d6879rgkjv   1/1       Running   0          11m
+svc-cat-etcd-5xtj4vlhx8                                   1/1       Running   0          1m
+svc-cat-etcd-chfwgmjdph                                   1/1       Running   0          47s
+svc-cat-etcd-jj87b2hmwg                                   1/1       Running   0          31s
+```
+
+You should also have an etcd service:
+
+```console
+$ kubectl get service
+NAME                          TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)             AGE
+etcd-restore-operator         ClusterIP   10.0.231.143   <none>        19999/TCP           14m
+kubernetes                    ClusterIP   10.0.0.1       <none>        443/TCP             56m
+svc-cat-etcd          ClusterIP   None           <none>        2379/TCP,2380/TCP   3m
+svc-cat-etcd-client   ClusterIP   10.0.2.237     <none>        2379/TCP            3m
+```
+
+The `svc-cat-etcd-client` service is what you will use to configure Service Catalog.
+
+### Install Service Catalog
+
+Once you have created an etcd cluster, it is time to install Service Catalog. You will use Helm to install Service Catalog. There are a few values you will need to override for your Service Catalog installation:
+
+* RBAC must be disabled (pending AKS support)
+* Embedded etcd must be disabled
+* You must point the installation at an external etcd.
```

Comment:
"external etcd" maybe should be called "an etcd cluster" so that people aren't thinking we are talking about an off-cluster ... etcd cluster.

p.s. cluster cluster cluster